rasa/core/processor.py

```diff
@@ -213,10 +215,39 @@ def get_tracker(self, conversation_id: Text) -> Optional[DialogueStateTracker]:
             conversation.
         """
         conversation_id = conversation_id or DEFAULT_SENDER_ID
```

Comment:
do we actually need this line? Or should it be `Optional[Text]` for `conversation_id`?

Reply:
I would keep it as it is in order to cover `conversation_id = ''`